source/global.h

```diff
@@ -10,16 +10,25 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 */
 
-#ifndef PI_H_
-#define PI_H_
+#ifndef GLOBAL_H_
+#define GLOBAL_H_
 
 
 
+const int FRAME_RATE = 60;
+
+const double DEG_0 = 0;
+const double DEG_90 = 90;
+const double DEG_180 = DEG_90 * 2;
```

Comment:
Why not simply 180?

Reply:
I meant to emphasize the geometrical tie between those constants (oops: `DG_0 = DEG_90 * 0`).

Reply:
Then again, why not `DEG_90 = DEG_360 / 4`? Given that this is a file for constants, the multiplication seems a bit out of place. Anyway, I suppose it's simply a matter of personal taste.

Reply:
Since they are resolved at compile time, I see no issue here, just a way to improve their consistency. Anyway, I agree with your criticism (see [Amendment 4](https://github.com/endless-sky/endless-sky/pull/4486/commits/c35cb7ec7a48ef55a385034307425337c3d426c9)).